cmd/controller/app/server.go

```diff
@@ -98,11 +103,32 @@ func Run(o *options.ControllerRunOptions, ctx context.Context) error {
 	clusterInformers := clusterv1informers.NewSharedInformerFactory(clusterClient, 10*time.Minute)
 	kubeInfomers := kubeinformers.NewSharedInformerFactory(kubeClient, 10*time.Minute)
 
+	// Cache only the LogCertSecret namespace secret
```

Comment:
can we have more comments here? does it mean if we want to watch other secret in the future, we will need to update the filter here?

Reply:
@qiujian16 yes, add more comments here.